webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -23,6 +23,58 @@
           </div>
           <div class="newsFormRightActions">
             <p class="draftSavingStatus">{{ draftSavingStatus }}</p>
+            <v-select
+              v-if="isCurentUserRedactor || isCurentUserSpaceManager"
```

Comment:
- I think that the selectBox will be displayed even for a non redactional space manager ?
- Have you checked if the selectBox is not displayed for a posted article draft ?